assemblyline/odm/models/result.py

```diff
@@ -179,6 +181,6 @@ def is_empty(self) -> bool:
         if len(self.response.extracted) == 0 and \
                 len(self.response.supplementary) == 0 and \
                 len(self.result.sections) == 0 and \
-                self.result.score == 0:
+                self.result.score == 0 and not self.partial:
```

Comment:
Why is partial having an impact on emptiness?

Reply:
IIRC empty results store and convey _nothing_ other than that a service ran, including whether the result was full or partial.